content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -180,6 +180,7 @@ Here are some examples of device plugin implementations:
     * Requires [nvidia-docker](https://github.com/NVIDIA/nvidia-docker) 2.0, which allows you to run GPU-enabled Docker containers.
 * The [NVIDIA GPU device plugin for Container-Optimized OS](https://github.com/GoogleCloudPlatform/container-engine-accelerators/tree/master/cmd/nvidia_gpu)
 * The [Intel device plugins](https://github.com/intel/intel-device-plugins-for-kubernetes) for Intel GPU, FPGA and QuickAssist devices
+* The [KubeVirt device plugins](https://github.com/kubevirt/kubernetes-device-plugins) for hardware-assisted virtualisation
```

Comment:
nit: If time, should the `Intel` and `KubeVirt` items be listed before NVIDIA?
small nit: virtualization?

Reply:
I've made some tweaks

>  line 55: what are `integer resources`?
I'll leave that line from the previous content. Another PR could fix that.